shell/aoe_translate.php

```diff
@@ -4,6 +4,12 @@
 
 class Aoe_Translate_Shell_Translate extends Mage_Shell_Abstract
 {
+    const EXPORT_TRANSLATION_FILE_NAME = 'aoe_translate.log';
+
+    /**
+     * @var array
+     */
+    protected $_exportFileHandles = [];
```

Comment:
Why do this?  The files are only ever used inside the export action. Making the lookup table live outside of that scope just makes the class more confusing.